src/Html/Builder.php

```diff
@@ -510,6 +510,24 @@ public function addIndex(array $attributes = [])
 
         return $this;
     }
+    
+    /**
+     * Reorderable
+     *
+     * @param       $column
```

Comment:
Can you pls add the missing type `string`? Sorry for overlooking this, haven't really able to test this but I think I will merge this since the code does look useful on your project.